it/mail/chrome/messenger/preferences/downloadactions.dtd

```diff
@@ -1,21 +1,21 @@
-<!ENTITY  downloadactionsWindow.title         "Download Actions">
+<!ENTITY  downloadactionsWindow.title         "Scarica azioni">
```

Comment:
CORRECTIONS NEEDED. To improve the consistence of the text, Azione/Azioni should be always written with a capital A. The correct entry should be as follows: 

+<!ENTITY  downloadactionsWindow.title         "Scarica Azioni">